third_party/coro/coro.c

```diff
@@ -103,13 +105,29 @@ coro_init (void)
 
   coro_transfer (new_coro, create_coro);
 
-#if __GCC_HAVE_DWARF2_CFI_ASM && __amd64
-  asm (".cfi_undefined rip");
-#endif
-
-  func ((void *)arg);
-
-  /* the new coro returned. bad. just abort() for now */
+  __asm__(
+	  /*
```

Comment:
Nit: Please use the code style of coro.c everywhere to be consistent (it means not tabs).

Reply:
Fixed: made code style consistent.